src/main/java/hudson/plugins/jacoco/ExecutionFileLoader.java

```diff
@@ -33,10 +34,10 @@
 		private String[] includes;
 		private String[] excludes;
 		
-		private ExecutionDataStore executionDataStore;
-		private SessionInfoStore sessionInfoStore;
+		private transient ExecutionDataStore executionDataStore;
```

Comment:
Can you add a comment why "transient" is necessary/useful here?

Reply:
Sure. It's because of the switch to org.jenkins-ci.plugins 3.0, which defines a FindBugs analysis.